### PR TITLE
Changelog cleanups

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: zcash_client_backend/CHANGELOG.md [hunk 1]
```diff
@@ -52,22 +42,53 @@ and this library adheres to Rust's notion of
...
-  with `NonNegativeAmount`.
-- `zcash_client_backend::wallet::WalletTransparentOutput::value` is now represented
-  as `NonNegativeAmount` instead of `Amount`, and constructors and accessors have been
-  updated accordingly.
-- `zcash_client_backend::wallet::ReceivedSaplingNote::value` is now represented
-  as `NonNegativeAmount` instead of `Amount`, and constructors and accessors have been
-  updated accordingly.
-- Almost all uses of `Amount` in `zcash_client_backend::zip321` have been replaced
-  with `NonNegativeAmount`.
+  - The error type of `wallet::create_spend_to_address` has been change to use
```
Comment (daira) [inline zcash_client_backend/CHANGELOG.md:45]:
```suggestion
  - The error type of `wallet::create_spend_to_address` has been changed to use
```

File: zcash_client_backend/CHANGELOG.md [hunk 2]
```diff
@@ -52,22 +42,53 @@ and this library adheres to Rust's notion of
...
+  - `ChangeValue::Sapling` is now a structured variant. In addition to the
+    existing change value, it now also carries an optional memo to be associated
+    with the change output.
+  - `fixed::SingleOutputChangeStrategy::new` and
+    `zip317::SingleOutputChangeStrategy::new` each now accept an additional
+    `change_memo` argument.
+- `zcash_client_backend::wallet`:
+  - The fields of `ReceivedSaplingNote` are now private. Use
+    `ReceivedSaplingNote::from_parts` for construction instead. Accessor methods
+    are provided for each previously-public field.
```
Comment (daira) [inline zcash_client_backend/CHANGELOG.md:64]:
```suggestion
    are provided for each previously public field.
```

File: zcash_primitives/CHANGELOG.md [hunk 3]
```diff
@@ -12,23 +12,39 @@ and this library adheres to Rust's notion of
...
-- Constants in `zcash_primitives::transaction::fees::zip317`:
-  - `MARGINAL_FEE`
-  - `GRACE_ACTIONS`
-  - `P2PKH_STANDARD_INPUT_SIZE`
-  - `P2PKH_STANDARD_OUTPUT_SIZE`
-- `zcash_primitives::transaction::builder::get_fee`
-- Additions related to `zcash_primitive::components::transaction::Amount`
-  and `zcash_primitive::components::transaction::NonNegativeAmount`
+- Additions related to `zcash_primitive::components::amount::Amount`
+  and `zcash_primitive::components::amount::NonNegativeAmount`
```
Comment (daira) [inline zcash_primitives/CHANGELOG.md:47]:
```suggestion
  and `zcash_primitive::components::amount::NonNegativeAmount`:
```

**Review by daira — DISMISSED** [review]
ACK with non-blocking suggestions. Doc-only PR.

**Comment by str4d** [comment]
[Force-pushed](https://github.com/zcash/librustzcash/compare/9c53fb94230c09ebaec83a92eb8825f47e64d629..8e0e77b0a0dbeb4ba77a1a457afbbe9211494f85) to address @daira's comments.

**Review by nuttycom — APPROVED** [review]
utACK with objections to all the busywork this is going to cause me.

**Review by daira — COMMENTED** [review]
Post-hoc ACK

**Pull request merged**